cmd/get_logs.go

```diff
@@ -296,7 +334,11 @@ func (glc *getLogsCmd) executeScript(node v1.Node, client *ssh.Client) (string,
 			cmd = fmt.Sprintf("bash -c \"export AZURE_ENV=%s; sudo -E %s\"", glc.getCloudName(), script)
 		}
 	} else {
-		script = "c:\\k\\debug\\collect-windows-logs.ps1"
+		if glc.windowsScriptPath != "" {
+			script = "(gi $env:temp).fullname + '\\collect-windows-logs.ps1'"
```

Comment:
You may be able to do `script = "$env:temp\\collect-windows-logs.ps1"` by adding `iex` as indicated below:

```
cmd = fmt.Sprintf("powershell -noprofile -command \"iex %s\"", filePath)
```

Reply:
Thanks, that works. Updated in code.